modules/REST/V2/Languages.php

```diff
@@ -0,0 +1,737 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\REST\V2;
+
+use PLL_Language;
+use PLL_Model;
+use stdClass;
+use WP_Error;
+use WP_REST_Controller;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Allows to access languages via the REST API.
+ *
+ * @since 3.7
+ *
+ * @phpstan-type Fields array{
```

Comment:
```suggestion
 * @phpstan-type EditableFields array{
```
Otherwise I might think it can be field to retrieve from a request. Like the `_filed=x` in WP routes.

Reply:
Done in [d36a21f](https://github.com/polylang/polylang/pull/1505/commits/d36a21f9dbdce3e81d4bb91085d9e9d18b0db310).